tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py{35,36,37,38}-sphinx{20,21,22,23}
+
+[testenv]
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
```

Comment:
One reason for the slower-than-usual response to this PR is that I had planned to switch away from Travis CI. That has now been done and this repository is on Github Actions.